hub/instruction_handling/create/_local.tex

```diff
@@ -13,6 +13,7 @@
 \def\locValueLo                                     {\col{STACK\_value\_lo}}
 \def\locIsStatic                                    {\col{current\_context\_is\_static}}
 \def\locStaticx                                     {\col{STACK\_staticx}}
+\def\locMaxcsx                                      {\col{STACK\_maxcsx}}
```

Comment:
`CREATE(2)` can now trigger the `maxCodeSizeException`, which previously could only be triggered by `RETURN`'s in a deployment context.